react/src/pages/participants/ParticipantInfoDialog.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useEffect } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { Dialog, DialogContent, Divider } from "@material-ui/core";
+import { ShowChart } from "@material-ui/icons";
+import { formatDateString, getAnalysisTitles, getAnalysisValues } from "../utils/functions";
+import { Participant, Analysis, Info } from "../utils/typings";
+import { DialogHeader } from "../utils/components/components";
+import SampleTable from "./SampleTable";
+import DetailSection from "../utils/components/DetailSection";
+import InfoList from "../utils/components/InfoList";
+
+const useStyles = makeStyles(theme => ({
+    dialogContent: {
+        margin: theme.spacing(0),
+        padding: theme.spacing(0),
+    },
+    infoSection: {
+        margin: theme.spacing(3),
+    },
+}));
+
+const getParticipantTitles = () => {
+    return [
+        "Participant ID",
+        "Family ID",
+        "Family Codename",
+        "Sex",
+        "Affected",
+        "Solved",
+        "Notes",
+        "Time of Creation",
+        "Created By",
+        "Time of Update",
+        "Updated By",
+    ];
+};
+const getParticipantValues = (participant: Participant) => {
+    return [
+        participant.participant_id,
+        participant.family_id,
+        participant.family_codename,
+        participant.sex,
+        participant.affected,
+        participant.solved,
+        participant.notes,
+        formatDateString(participant.created),
+        participant.created_by,
+        formatDateString(participant.updated),
+        participant.updated_by,
+    ];
+};
+function getAnalysisInfoList(analyses: Analysis[]) {
+    return analyses.map(analysis => {
+        return {
+            primaryListTitle: `Analysis ID ${analysis.analysis_id}`,
+            secondaryListTitle: `Current State: ${analysis.analysis_state} - Click for more details`,
+            titles: getAnalysisTitles(),
+            values: getAnalysisValues(analysis),
+        } as Info;
```

Comment:
Typo the return value of the function as `Info[]` instead of performing a type cast